_includes/announce.html

```diff
@@ -7,13 +7,9 @@ <h2 class="dark">Last chance - Apply now!</h2>
    </div>
    <div class="col12 clearfix contain center space-bottom8">
       <div class="limiter">
-         <div class="col6 pad0y space-bottom2">
-            <h3 class="announce-title"><a href="https://blog.openstreetmap.org/2017/02/16/propose-your-session-to-state-of-the-map-2017/" class="sub_color">Call for Proposals</a></h3>
-            <p class="varela">Please submit your session proposals for a chance to be included in the program.</p>
-         </div>
-         <div class="col6 pad0y space-bottom2">
-	         <h3 class="announce-title"><a href="https://blog.openstreetmap.org/2017/02/22/apply-for-a-scholarship-to-state-of-the-map-2017/" class="sub_color">Apply for a Scholarship</a></h3>
-            <p class="varela">Thanks to the support of our sponsors, scholarships will help bring us together.</p>
```

Comment:
<div class="col12 clearfix contain center space-bottom8">
      <div class="limiter">
         <div class="col6 pad0y space-bottom2">
            <h3 class="announce-title">Conference program</h3>
            <p class="varela">Thank you for the great responses to our <a href="https://blog.openstreetmap.org/2017/02/16/propose-your-session-to-state-of-the-map-2017/" class="sub_color">call for proposals</a> and <a href="https://blog.openstreetmap.org/2017/04/11/community-survey-for-the-state-of-the-map-2017-program/" class="sub_color">community input</a> for building the program!</p>
         </div>
         <div class="col6 pad0y space-bottom2">
            <h3 class="announce-title">Scholarship program</h3>
            <p class="varela">Thanks to the support of our sponsors, scholarships will help bring us together. Thank you to all the great people <a href="https://blog.openstreetmap.org/2017/02/22/apply-for-a-scholarship-to-state-of-the-map-2017/" class="sub_color">who applied for a scholarship</a>!</p>
         </div>
      </div>
   </div>